requirements/base.txt

```diff
@@ -1,4 +1,3 @@
-django-import-export==1.0.0
-openpyxl==2.4.9
-django-admin-rangefilter==0.2.1
-qrcode==6.0
+django-admin-rangefilter==0.5.1
+django-import-export==1.2.0
```

Comment:
I remember we had a problem with the version of **openpyxl**, reason why we pinned it to 2.4.9. did it get already resolved to remove the dependency?

Reply:
I have not tested this and I don't know which was the problem with openpyxl

Reply:
It seems it was fixed [here](https://github.com/jazzband/tablib/issues/324), anyways I will try to test everything's OK